src/components/TodoApp.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useContext } from 'react';
+import { Header } from './Header';
+import { Main } from './Main';
+import { TodoContext } from '../TodoContext';
+import { TodosFilter } from './TodosFilter';
+
+export const TodoApp: React.FC = () => {
+  const { todos } = useContext(TodoContext);
+
+  return (
+    <div className="todoapp">
+      <Header />
+      {todos.length > 0
+      && (
```

Comment:
```suggestion
      {todos.length && (
```
Can be simplified

Reply:
I couldn’t do exactly as you advised (I wrote more in detail in my personal channel), but I tried to simplify it.